hack/verify-release.sh

```diff
@@ -213,7 +213,7 @@ check_tools()
         case "${tool}" in
             osv-scanner)
                 version=$("${OSVSCANNER_CMD[@]}" -v | grep version | cut -f3 -d" ")
-                min_version="2.0.0"
+                min_version="2.1.0"
```

Comment:
FYI: Min version doesn't have to be bumped unless old version is actually incompatible, ie. different flags etc. It was done for v1 vs v2 being different. But since this same bump is applied to other repos, let's take it in as is.